### ISSUE TITLE
Issue #100778 closed due to inactivity of Flutter team members

### ISSUE THREAD

**Issue by koral--** [flutter/flutter#101704]
> I don't think this issue is different from https://github.com/flutter/flutter/issues/100777 apart from `Another exception was thrown: Instance of 'DiagnosticsProperty<void>` exception. Would be good to track this in already triaged and linked issue.
> Closing this as duplicate. If you disagree, write in comments and I'll reopen it.

_Originally posted by @darshankawar in https://github.com/flutter/flutter/issues/100778#issuecomment-1080641191_

I've responded here: https://github.com/flutter/flutter/issues/100778#issuecomment-1080737799 but @darshankawar haven't reacted until issues gets closed by bot.

I could not post comments in a closed issue hence I'm creating a new ticket, intentionally omitting a template. After #100778 gets reopened this one can be closed.

**Comment by darshankawar** [comment]
Thanks @koral-- for pointing this. The linked issue somehow got lost in the notifications.
I'll reopen it and closing this.
